github/pkg/reconciler/githubsource.go

```diff
@@ -160,19 +160,21 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, source *sourcesv1alpha1.
 
 		// source.Status.MarkServiceDeployed(ra)
 		// TODO: Mark Deployed for the ksvc
-		// TODO: Mark some condition for the webhook status?
 		if source.Status.WebhookIDKey == "" {
 			hookID, err := r.createWebhook(ctx, args)
 			if err != nil {
+				source.Status.MarkNoWebhook("WebhookCreationFailed", "%s", err)
```

Comment:
nit: the "Webhook" here feels redundant with the Type.
```suggestion
				source.Status.MarkNoWebhook("CreationFailed", err.Error())
```